package.json

```diff
@@ -11,14 +11,15 @@
     "webpack": "webpack",
     "test": "mocha -r ts-node/register tests/**/*Tests.ts",
     "coverage": "nyc --require ts-node/register --reporter=lcov --reporter=text npm run test",
-    "lint": "tslint -c tslint.json 'src/**/*.ts'"
+    "lint": "tslint -c tslint.json 'src/**/*.ts'",
+    "posttest": "nyc report --reporter=json"
```

Comment:
This posttest target will be run automatically on travis - see [the docs](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/)